service_discovery/dto/factory_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package dto
+
+import (
+	"testing"
+
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	providerID      = identity.FromAddress("123456")
+	providerContact = Contact{
+		Type: "type1",
+	}
+)
+
+func Test_NewServiceProposal(t *testing.T) {
```

Comment:
does this really test the New proposal? Is it not update?

Reply:
Fixed